### PR TITLE
Feature/pdq cis

### PR DESCRIPTION
**Pull request by bryanpnih**
This is the Review PR for PDQ CIS look and feel into develop.

Individual developer changes should be PR'd to the feature/pdq-cis branch of NCIOCPL.

### CODE REVIEW

**Comment by bryanpnih** [comment]
http://ncigovcdode36.prod.acquia-sites.com/

**Comment by VictoriaSunNIH** [comment]
@bryanpizzillo , @alifr , @lburack - what Nadia has found so far:
For http://ncigovcdode36.prod.acquia-sites.com/about-cancer/treatment/cam/hp/cartilage-pdq
1. Terms should not be capitalized in the modal windows
2. On mobile, On This Page appears above the accordion (and should not appear.  @alifr noted this was an issue already in his ticket #739 )
3. Too much space between the h2 and content (compared to prod)
4. Too much space above h3s under About This Summary
5. Page options are not appearing on PDQ summaries

On http://ncigovcdode36.prod.acquia-sites.com/types/breast/hp/breast-treatment-pdq
1. Inconsistency in spacing between  ‘a’ tag elements when ‘a’ elements are indented under In This Section
2. In Situ in Ductal Carcinoma In Situ appears within em tag (under On This Page and when it appears as h2
3. More space has been introduced above the h3s (Anatomy, Protective Factors etc.)
4. The content in the table footer is appearing bold (font weight 700)
5. The font weight of PDQ list title under Treatment Option Overview for Early/Localized/Operable Breast Cancer appears as 400 whereas in Prod it is 700 italic.
6. More space above h4 (Axillary lymph node management)
7. More space between h2 and In This Section 

Generally on IE:
1..	IE fonts looks different
2.	Modals are not appearing on IE - Getting 404 page on clicking modal
3.	Citation jump links do not work in IE

**Comment by VictoriaSunNIH** [comment]
See comments on #1183

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/pdq/PDQ.js [hunk 1]
```diff
@@ -19,8 +19,11 @@ document.addEventListener('DOMContentLoaded',onDOMContentLoaded);
 /* TODO: create a field in the content type for 'hp-patient-toggle-link' */
 const moveToggle = () => {
   const toggle = document.querySelector('.pdq-hp-patient-toggle', '#cgvBody');
```
Comment (BrendanBeltzNIH) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/pdq/PDQ.js]: As far as I know, the api for document.querySelector does not handle more than one argument. I see it being used with multiple params in a couple places here. I believe the 2+ arguments are simply being ignored.

Comment (alifr) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/pdq/PDQ.js]: You're correct. This should be one string argument with CSS selectors separated by a comma `document.querySelector('.pdq-hp-patient-toggle, #cgvBody')`

Comment (alifr) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/pdq/PDQ.js]: Correction, that second argument is intended to provide context to the selector the way jQuery does it. It should be `document.querySelector('#cgvBody .pdq-hp-patient-toggle')`

**Comment by VictoriaSunNIH** [comment]
Passes IA Review.  Remaining issues are global (spacing on the public use text display, and issues with the modals for glossified terms)

**Pull request merged**